test/tomo/operators/ray_trafo_test.py

```diff
@@ -155,7 +155,14 @@ def projector(request):
         raise ValueError('geom not valid')
 
 
-def test_projector(projector):
+@pytest.fixture(scope="module",
+                params=[True, False],
+                ids=[' inplace ', ' out of place '])
```

Comment:
I would prefer "in place" or "in-place" since "inplace" does not seem to be a proper English word.